components/public-api-server/pkg/server/server.go

```diff
@@ -126,6 +126,11 @@ func Start(logger *logrus.Entry, version string, cfg *config.Configuration) erro
 		log.Info("No Personal Access Token signign key specified, PersonalAccessToken service will be disabled.")
 	}
 
+	_, err = auth.NewJWTFromAuthPKI(cfg.Auth.PKI, 7*24*time.Hour, "TODO")
```

Comment:
For now, we don't need to set the issuer as we're not actually issuing any JWTs. 

What we'll want to do here is match the issuer on "server" - gitpod domain.

Reply:
To supply consistent values, there's https://github.com/gitpod-io/gitpod/pull/17314 which will use the same config between server/public-api